spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -1509,23 +1501,20 @@ def setup_firefox_with_linux
       }
       @tb_buttons = {}
       @item_out = {}
-      allow_any_instance_of(Object).to receive(:query_string).and_return("")
```

Comment:
As bad as the public all the private methods workaround is... removing any_instance_of mocks on Object makes it all worthwhile.